bin/dfx-sns-demo

```diff
@@ -74,8 +74,6 @@ sleep 1
 bin/dfx-neuron-prolong --network "$DFX_NETWORK"
 sleep 1
 
```

Comment:
Now whitelisting is valid for only one SNS, so instead of being run just once at setup, needs to be run for every SNS.